kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -93,6 +93,18 @@ public class BigQuerySinkConfig extends AbstractConfig {
       "The interval, in seconds, in which to attempt to run GCS to BQ load jobs. Only relevant "
       + "if enableBatchLoad is configured.";
 
+  public static final String BQ_STREAMING_MAX_ROWS_PER_REQUEST_CONFIG =                     "bqStreamingMaxRowsPerRequest";
+  private static final ConfigDef.Type BQ_STREAMING_MAX_ROWS_PER_REQUEST_TYPE =              ConfigDef.Type.INT;
+  private static final Integer BQ_STREAMING_MAX_ROWS_PER_REQUEST_DEFAULT =                  50000;
```

Comment:
Let's have the default behaviour same. We can use '-1' to say this is disabled and have that as the default